project_generation/content/templates/controller/config/config_test.go.tmpl

```diff
@@ -9,31 +9,26 @@ import (
 )
 
 func TestConfig(t *testing.T) {
-	os.Clearenv()
-	var err error
-	var configuration *Config
-	
 	Convey("Given an environment with no environment variables set", t, func() {
-		Convey("Then cfg should be nil", func() {
-			So(cfg, ShouldBeNil)
-		})
+		os.Clearenv()
+		cfg, err := Get()
 
 		Convey("When the config values are retrieved", func() {
 
-			Convey("Then there should be no error returned, and values are as expected", func() {
-				configuration, err = Get() // This Get() is only called once, when inside this function
+			Convey("Then there should be no error returned", func() {
 				So(err, ShouldBeNil)
-				So(configuration, ShouldResemble, &Config{
-					BindAddr:                   ":{{.Port}}",
-					GracefulShutdownTimeout:    5 * time.Second,
-					HealthCheckInterval:        30 * time.Second,
-					HealthCheckCriticalTimeout: 90 * time.Second,
-					HelloWorldEmphasise:        true,
-				})
+			})
+
+			Convey("Then the values should be set to the expected defaults", func() {
+				So(cfg.BindAddr, ShouldEqual, ":{{.Port}}")
+				So(cfg.GracefulShutdownTimeout, ShouldEqual, 5*time.Second)
+				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
+				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
+				// TODO: Remove assertion once configuration variable has been removed from config.go
```

Comment:
did you mean to leave this TODO ?

Reply:
Yes, these TODOs are added deliberately so that they appear in the generated projects 👍